shell/config/router.js

```diff
@@ -6,375 +6,396 @@ import scrollBehavior from '../utils/router.scrollBehavior.js';
 
-export const linkActiveClass = 'nuxt-link-active';
-
 Vue.use(Router);
 
 export const routerOptions = {
-  mode:                 'history',
+  mode: 'history',
   // Note: router base comes from the ROUTER_BASE env var
-  base:                 process.env.routerBase || '/',
-  linkActiveClass,
-  linkExactActiveClass: 'nuxt-link-exact-active',
+  base: process.env.routerBase || '/',
   scrollBehavior,
 
-  routes: [{
-    path:      '/about',
-    component: () => interopDefault(import('../pages/about.vue' /* webpackChunkName: "pages/about" */)),
-    name:      'about'
-  }, {
-    path:      '/account',
-    component: () => interopDefault(import('../pages/account/index.vue' /* webpackChunkName: "pages/account/index" */)),
-    name:      'account'
-  }, {
-    path:      '/c',
-    component: () => interopDefault(import('../pages/c/index.vue' /* webpackChunkName: "pages/c/index" */)),
-    name:      'c'
-  }, {
-    path:      '/clusters',
-    component: () => interopDefault(import('../pages/clusters/index.vue' /* webpackChunkName: "pages/clusters/index" */)),
-    name:      'clusters'
-  }, {
-    path:      '/diagnostic',
-    component: () => interopDefault(import('../pages/diagnostic.vue' /* webpackChunkName: "pages/diagnostic" */)),
-    name:      'diagnostic'
-  }, {
-    path:      '/fail-whale',
-    component: () => interopDefault(import('../pages/fail-whale.vue' /* webpackChunkName: "pages/fail-whale" */)),
-    name:      'fail-whale'
-  }, {
-    path:      '/home',
-    component: () => interopDefault(import('../pages/home.vue' /* webpackChunkName: "pages/home" */)),
-    name:      'home'
-  }, {
-    path:      '/prefs',
-    component: () => interopDefault(import('../pages/prefs.vue' /* webpackChunkName: "pages/prefs" */)),
-    name:      'prefs'
-  }, {
-    path:      '/support',
-    component: () => interopDefault(import('../pages/support/index.vue' /* webpackChunkName: "pages/support/index" */)),
-    name:      'support'
-  }, {
-    path:      '/account/create-key',
-    component: () => interopDefault(import('../pages/account/create-key.vue' /* webpackChunkName: "pages/account/create-key" */)),
-    name:      'account-create-key'
-  }, {
-    path:      '/auth/login',
-    component: () => interopDefault(import('../pages/auth/login.vue' /* webpackChunkName: "pages/auth/login" */)),
-    name:      'auth-login'
-  }, {
-    path:      '/auth/logout',
-    component: () => interopDefault(import('../pages/auth/logout.vue' /* webpackChunkName: "pages/auth/logout" */)),
-    name:      'auth-logout'
-  }, {
-    path:      '/auth/setup',
-    component: () => interopDefault(import('../pages/auth/setup.vue' /* webpackChunkName: "pages/auth/setup" */)),
-    name:      'auth-setup'
-  }, {
-    path:      '/auth/verify',
-    component: () => interopDefault(import('../pages/auth/verify.vue' /* webpackChunkName: "pages/auth/verify" */)),
-    name:      'auth-verify'
-  }, {
-    path:      '/rio/mesh',
```

Comment:
I've seen that you've deleted the `rio/mesh` page as well. What was this used for?

Reply:
It was really the basis for fleet but is no longer used https://github.com/rancher/rio